src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -2584,7 +2615,7 @@ public void TestNegUndefWithBadNumericalName()
             var node = Parse(text);
             TestRoundTripping(node, text, false);
             VerifyErrorCode(node, (int)ErrorCode.ERR_IdentifierExpected);
-            VerifyDirectivesSpecial(node, new DirectiveInfo { Kind = SyntaxKind.UndefDirectiveTrivia, Status = NodeStatus.IsActive, Text = "1234" });
+            VerifyDirectivesSpecial(node, new DirectiveInfo { Kind = SyntaxKind.UndefDirectiveTrivia, Status = NodeStatus.IsActive, Text = "" });
```

Comment:
Note: the `1234 /// ...` is parsed as part of EndOfDirective, not the Name.
This one does seem strange. Let me know if I should file a bug.